bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/RuleEventConstants.java

```diff
@@ -0,0 +1,156 @@
+/*******************************************************************************
+ * Copyright (c) 1997, 2015 by ProSyst Software GmbH
+ * http://www.prosyst.com
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    ProSyst Software GmbH - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.smarthome.automation;
+
+import java.util.Collection;
+
+/**
+ * This class defines event topics and event properties which are used by the
+ * automation events.
+ *
+ * @author Yordan Mihaylov - Initial Contribution
+ */
+public interface RuleEventConstants {
```

Comment:
It seems that these constants are not yet used in the implementation, right?
Regarding eventing, please not that we will soon contribute an implementation of what we introduced here: https://www.eclipse.org/forums/index.php/t/1066185/
The automation events should be in line with this new infrastructure, especially regarding used topic structure etc.